keras_cv/src/models/object_detection/mask_rcnn/mask_head.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2024 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.src.api_export import keras_cv_export
+from keras_cv.src.backend import keras
+
+
+@keras_cv_export(
+    "keras_cv.models.mask_rcnn.MaskHead",
+    package="keras_cv.models.mask_rcnn",
+)
+class MaskHead(keras.layers.Layer):
+    """A Keras layer implementing the R-CNN Mask Head.
+
+    The architecture is adopted from Matterport's Mask R-CNN implementation
+    https://github.com/matterport/Mask_RCNN/blob/master/mrcnn/model.py.
+
+    Args:
+        num_classes: The number of object classes that are being detected.
+        conv_dims: (Optional) a list of integers specifying the number of
```

Comment:
if we are fllowing conventions from other models This should be renamed to `stackwise_num_conv_filters` and  `stackwise_num_deconv_filters`

Reply:
I have changed to `stackwise_num_conv_filters` and `num_deconv_filters`. stackwise doesn't make sense for `num_deconv_filters` since there is no stack.